### PR TITLE
FABJ-536 Port changes to master

### PR DESCRIPTION
**Pull request by yzhivkov**
…t to use Hostname verifier in test mode

Signed-off-by: TsvetanG <tsvetan.georgiev@gmail.com>

Add info messages in the output when running the test

Signed-off-by: TsvetanG <tsvetan.georgiev@gmail.com>

FABJ-536 code review fixes

Signed-off-by: TsvetanG <tsvetan.georgiev@gmail.com>
Signed-off-by: Yanko Zhelyazkov <yanko@senofi.ca>

### CODE REVIEW

File: src/test/java/org/hyperledger/fabric/sdkintegration/NetworkConfigIT.java [hunk 1]
```diff
@@ -169,6 +171,27 @@ public static void doMainSetup() throws Exception {
         mockuser.setEnrollment(hfcaClient.enroll(mockuser.getName(), mockuser.getEnrollmentSecret()));
         orgRegisteredUsers.put(org.getName(), mockuser);
 
+//        if (testConfig.isRunningFabricTLS()) {
```
Comment (bestbeforetoday) [inline src/test/java/org/hyperledger/fabric/sdkintegration/NetworkConfigIT.java:174]: This line isn't commented out in your cherry-pick to _release-2.2_, and that build looks to have passed

Comment (yzhivkov) [reply src/test/java/org/hyperledger/fabric/sdkintegration/NetworkConfigIT.java:174]: Please disregard the two pull requests to master and release-2.2 for now. Once they are ready I will move them from draft status to ready to review. We are currently having trouble with the integration test

**Comment by J-Ankit2020** [comment]
Hi sir,
I am Ankit and I want to do my first open source contribution .Shall I do my first contribution here. I know java and I am here to learn new things. It will be a great pleasure if you say yes

File: src/test/java/org/hyperledger/fabric/sdkintegration/NetworkConfigIT.java (hunk 1 above)
Comment (yzhivkov) [reply src/test/java/org/hyperledger/fabric/sdkintegration/NetworkConfigIT.java]: The PR is ready for code review

File: src/main/java/org/hyperledger/fabric/sdk/NetworkConfig.java [hunk 2]
```diff
@@ -973,7 +1000,7 @@ private CAInfo createCA(JsonObject jsonCA, OrgInfo org) throws NetworkConfigurat
             }
         }
 
-        CAInfo caInfo = new CAInfo(org.mspId, url, regUsers, httpOptions);
+        CAInfo caInfo = new CAInfo(name, url, regUsers, httpOptions);
```
Comment (bestbeforetoday) [inline src/main/java/org/hyperledger/fabric/sdk/NetworkConfig.java:1003]: Is this change necessary?

Comment (yzhivkov) [reply src/main/java/org/hyperledger/fabric/sdk/NetworkConfig.java:1003]: There was no easy way to pull the CA that we need for the integration tests otherwise.

**Review by bestbeforetoday — APPROVED** [review]

**Comment by bestbeforetoday** [comment]
The change needs to be rebased on the current master branch before it can be merged
